site/_includes/header.html

```diff
@@ -27,6 +27,9 @@
             </div>
           </form>
           <ul class="nav navbar-nav navbar-right">
+            <li{% if page.nav == "docs" %} class="active"{% endif %}>
```

Comment:
This adds a top level navbar item to the `site.getting_started_category_url` variable set in `_config.yml`.